api/alembic/versions/004ac48ffe18_indices.py

```diff
@@ -0,0 +1,83 @@
+"""indices
+
+Revision ID: 004ac48ffe18
+Revises: 81c96876355a
+Create Date: 2021-03-29 18:24:10.485482
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '004ac48ffe18'
+down_revision = '81c96876355a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_index(op.f('ix_hourly_actuals_rh_valid'), 'hourly_actuals', ['rh_valid'], unique=False)
+    op.create_index(op.f('ix_hourly_actuals_station_code'), 'hourly_actuals', ['station_code'], unique=False)
+    op.create_index(op.f('ix_hourly_actuals_temp_valid'), 'hourly_actuals', ['temp_valid'], unique=False)
```

Comment:
why is it only `temp_valid` and `rh_valid` that are getting indices? Why not the `_valid`s for wind speed & dir, precip, etc.?